Add troubleshooting info for version discovery on unsupported Linux platforms (#2749)
sensu/sensu-docs: content/sensu-go/6.0/operations/maintain-sensu/troubleshoot.md

This would allow the asset to be downloaded onto the target entity.

### Running the agent on an unsupported Linux platform

If you run the Sensu agent on an unsupported Linux platform, the agent might fail to correctly identify your version of Linux and could download the wrong version of an asset.

> This issue is specific to Linux distributions that do not include the `lsb_release` package in their default installations.

Author: ```suggestion
This issue affects Linux distributions that do not include the `lsb_release` package in their default installations.
```
